contracts/governance/FeeSharingCollector/FeeSharingCollector.sol

```diff
@@ -393,35 +406,80 @@ contract FeeSharingCollector is
      *
      * @dev WARNING! This function skips all the checkpoints before '_fromCheckpoint' irreversibly, use with care
      *
-     * @param _token RBTC dummy to fit into existing data structure or SOV. Former address of the pool token.
-     * @param _fromCheckpoint Skips all the checkpoints before '_fromCheckpoint'
+     * @param _tokens RBTC dummy to fit into existing data structure or SOV. Former address of the pool token.
+     * @param _fromCheckpoints Skips all the checkpoints before '_fromCheckpoint'
      *        should be calculated offchain with getNextPositiveUserCheckpoint function
      * @param _maxCheckpoints Maximum number of checkpoints to be processed.
      * @param _receiver The receiver of tokens or msg.sender
      * */
     function withdrawStartingFromCheckpoint(
+        address[] calldata _tokens,
```

Comment:
intended to process multiple tokens at once, but on the unit tests, this function is never tested with more than 1 token address (always SOV). Please add a test where multiple assets are being withdrawn, especially also rbtc tokens.

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/507/commits/695fa69d802d4f5e9417b4a0a8ca9acee1977b98)